.appveyor.yml

```diff
@@ -16,47 +16,49 @@ environment:
     CYG_ROOT: C:\cygwin64
 
   matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2022
```

Comment:
Surprising that there is still appveyor file here, we removed them from almost all the repos